internal/server/server.go

```diff
@@ -157,7 +149,7 @@ func New(options Options) (*Server, error) {
 		return nil, fmt.Errorf("loading certificate provider: %w", err)
 	}
 
-	settings, err := data.InitializeSettings(server.db)
+	_, err = data.InitializeSettings(server.db)
```

Comment:
we should update `InitializeSettings` to not return a settings struct. It isn't used anymore as of this change.